platform/ios/src/MGLMapView.mm

```diff
@@ -378,7 +377,7 @@ - (void)setStyleURL:(nullable NSURL *)styleURL
     {
         styleURL = [MGLStyle streetsStyleURLWithVersion:MGLStyleDefaultVersion];
     }
-
+    [[MGLLogging sharedInstance] debug:[NSString stringWithFormat:@"%s", __PRETTY_FUNCTION__] message:@"styleURL: %@", styleURL];
```

Comment:
It’s inconvenient to convert a C string to an NSString each time something needs to be logged. If we always expect `__PRETTY_FUNCTION__` to be used here, consider having the method take a C string as its `fileName` parameter or, better yet, define a macro that inserts `__PRETTY_FUNCTION__` automatically.